4.plot_mutation_results.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plot results of mutation detection experiments"
```

Comment:
Adding a description here would be helpful for understanding the notebook. In particular, it is unclear why you picked your set of genes (though that may just be because I don't know cancer genes other than TP53 and BRCA).